rust/lance-tools/src/cli.rs

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright The Lance Authors
+
+use clap::{Parser, Subcommand};
+use std::io::Error;
+use std::io::ErrorKind;
+use lance_core::Result as LanceResult;
+
+#[derive(Parser, Debug)]
+#[command(
+    name = "lance-tools",
+    about = "Tools for working with Lance files",
```

Comment:
I think it would be nice if the command can eventually cover both file and table, so we can have a subcommand domain like `lance file meta -s /path/to/file`, so that we can have another table domain like `lance table -s /path/to/table`

Reply:
Agreed actually.  I often need to ask users to print out and send me their manifest when I am debugging issues and right now I have to do this by asking them to run a python script...

```
import lance
from lance.debug import format_manifest

ds = lance.dataset(...)
print(format_manifest(ds))
```

Having a CLI tool that can print the manifest would be pretty handy.